manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml

```diff
@@ -0,0 +1,286 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federation-v2.v0.0.4
+  namespace: placeholder
```

Comment:
Are there any examples for federation CRs? If so these can be added into `metadata.annotations.alm-examples` as stringified JSON.

Reply:
I'll point you at some per our offline convo